Unity/Assets/YarnSpinner/Editor/YarnImporterEditor.cs

```diff
@@ -160,4 +173,229 @@ public class YarnImporterEditor : ScriptedImporterEditor {
 
         AssetDatabase.ImportAsset(assetPath);
     }
+
+    void UpdateLocalizations(string newBaseCsv)
```

Comment:
The documentation is really comprehensive in the second half of `MergeStringTables`, but there's a bit less in the first half and in `UpdateLocalizations`. Can you expand on the comments a bit?

Reply:
Yep, new commit up with more comments and fixes